practice-project-api/routes/calculateRoutes.ts

```diff
@@ -0,0 +1,39 @@
+import { Router, Request, Response } from "express";
+
+export const calculateRoute = Router();
+
+calculateRoute.post("/calculate", (req: Request, res: Response, next) => {
+
+  enum calculatorOperation {
```

Comment:
Also, all of these should be outside the post function.

Reply:
The only thing you want inside here is the handling of the requets, everything else should be declared outside

Reply:
Enums should be pascal case rather than camel case.